cmd/minikube/cmd/node_add.go

```diff
@@ -51,6 +51,11 @@ var nodeAddCmd = &cobra.Command{
 		name := node.Name(len(cc.Nodes) + 1)
 
 		out.Step(style.Happy, "Adding node {{.name}} to cluster {{.cluster}}", out.V{"name": name, "cluster": cc.Name})
+		// for now control-plane feature is not supported
+		if !cp {
+			out.Step(style.Unsupported, "control-plane feature is not supported {{.name}} to {{.cluster}}!", out.V{"name": name, "cluster": cc.Name})
```

Comment:
```suggestion
			out.Step(style.Unsupported, "Adding a control-plane node is not yet supported, setting control-plane flag to false")
```

Could you also move this `if` statement above the `out.Step(style.Happy, "Adding node` line.